### DISCUSSION TITLE
Newbie question to use a model for luxemburgish langauge

### DISCUSSION THREAD

**Discussion by jevylux** [openai/whisper#1968] [Q&A]
Hi, I am new to all of this, but I want to use whisper with a model un luxemburgish language.
I have downloaded some files, but now I am stuck.
the files are the following
<img width="767" alt="Screenshot 2024-01-20 at 15 20 47" src="https://github.com/openai/whisper/assets/198311/1531785b-691d-4170-ae8f-73e9f4896efb">

Now, what exactly should i do ? I want to use either rhasspy 3  or some python programm

What do I have to so ? Is there a tutorial somewhere ?

**Comment by phineas-pta** [comment]
seem like those files are for huggingface `transformers` package, u can start from there
